src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OtlpLogRecordTransformer.cs

```diff
@@ -16,6 +16,8 @@ namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
 
 internal sealed class OtlpLogRecordTransformer
 {
+    private const string DefaultScopeName = "unknown_scope";
```

Comment:
Is this part of the spec. Why cannot we put logs without logger name individually under a new scope?

Reply:
Probably a more sensible default is just `string.Empty`. If you request a logger without supplying a name (eg `loggerProvider.GetLogger()`), that's [what you'll get](https://github.com/open-telemetry/opentelemetry-dotnet/blob/8b54074c7a392b0e295fa7aa0f1a3ab1cb28025a/src/OpenTelemetry.Api/Logs/Logger.cs#L37). In practice we should never run into this default case as there will always be a `Logger` either via ILogger path or Log Bridge API. But good to be defensive here I think.

Reply:
Empty sounds reasonable to me.